modules/cluster/addons/aws-k8s-cni.yaml

```diff
@@ -86,15 +99,27 @@ spec:
       tolerations:
         - operator: Exists
       containers:
-        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.5.7
+        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.6.1
           imagePullPolicy: Always
           ports:
             - containerPort: 61678
               name: metrics
           name: aws-node
+          readinessProbe:
+            exec:
+              command: ["/app/grpc-health-probe", "-addr=:50051"]
+            initialDelaySeconds: 35
+          livenessProbe:
+            exec:
+              command: ["/app/grpc-health-probe", "-addr=:50051"]
+            initialDelaySeconds: 35
           env:
             - name: AWS_VPC_K8S_CNI_LOGLEVEL
               value: DEBUG
+            - name: AWS_VPC_K8S_CNI_VETHPREFIX
+              value: eni
+            - name: AWS_VPC_ENI_MTU
+              value: "9001"
```

Comment:
Why are we defining this value, given that we're using the default value anyway? https://github.com/aws/amazon-vpc-cni-k8s/blob/501ca48d682670c45e8303c631cec8e178db6c89/pkg/networkutils/network.go#L935

Maybe it would be better to have us use the default value, which presumably would be the recommended value for what is the most common setup (linux, amd64) for this tool?

Reply:
I think setting the default here is OK ... it makes it more explicit so we can see what it is without checking the sourcecode.

Also it's best not to change the yaml from the upstream if we can help it ... it makes updates simpler!

Reply:
Oh I didn't realize this was the upstream manifest. Cool let's keep it. :smiley: